### PR TITLE
Make the UA and navigator.appVersion match.

### PR DESCRIPTION
**Pull request by sayrer**
Seems to work, lmk if there should be a test somewhere.

Screen Shot 2022-06-09 at 2 36 38 PM

### CODE REVIEW

File: Userland/Libraries/LibWeb/Loader/ResourceLoader.h [hunk 1]
```diff
@@ -23,7 +23,10 @@ namespace Web {
 #    define CPU_STRING "x86_64"
 #endif
 
-constexpr auto default_user_agent = "Mozilla/5.0 (SerenityOS; " CPU_STRING ") LibWeb+LibJS/1.0 Browser/1.0";
+#define APP_VERSION "5.0"
+
+constexpr auto app_version = APP_VERSION;
+constexpr auto default_user_agent = "Mozilla/" APP_VERSION " (SerenityOS; " CPU_STRING ") LibWeb+LibJS/1.0 Browser/1.0";
```
Comment (linusg) [inline Userland/Libraries/LibWeb/Loader/ResourceLoader.h:29]: This is overkill, please revert. We're not ever going to bump this version anyway.

**Review by linusg — CHANGES_REQUESTED** [review]

File: Userland/Libraries/LibWeb/Loader/ResourceLoader.h (hunk 1 above)
Comment (sayrer) [reply Userland/Libraries/LibWeb/Loader/ResourceLoader.h:29]: Sure, do you just want two "5.0" literals?

Comment (linusg) [reply Userland/Libraries/LibWeb/Loader/ResourceLoader.h:29]: Right :)

**Review by linusg — COMMENTED** [review]
and now please squash both commits into one as per the [contribution guidelines](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy) :)

**Comment by sayrer** [comment]
I closed this and gave up:

1) the strategy of just using a few duplicated constants is fine, unless you actually get a bug like this. Then you should fix the duplication.

2) there is a button in the GitHub UI to squash the commit if you want

I wish you the best with your project.

**Comment by linusg** [comment]
> there is a button in the GitHub UI to squash the commit if you want

We disabled that after accidental quashes of nicely prepared sets of multiple atomic commits.
The burden of ensuring that contributions adhere to the contribution guidelines is on each individual contributor, not the maintainers - that's just how it is.

**Comment by sayrer** [comment]
Well, feel free to copy/paste my fix. btw—an emdash (with no spaces) is how you type that. I'd say it's useless pedantry, but you get my drift.

**Pull request closed without merging**